### ISSUE TITLE
dcterms:isVersionOf not repeatable in ERA

### ISSUE THREAD

**Issue by anayram** [ualbertalib/HydraNorth#1037]
While testing oai I found that the field dcterms:isVersionOf, which is repeatable in old ERA/thesisdeposit, is not repeatable in new ERA.

@weiweishi @pbinkley do you know if the data was concatenated into a single field in Fedora (which would require adding to the list of audit changes) or if it is only a problem with the way data is presented in the public view form?

Example: the item below comes with multiple isVersionOf fields but data is lumped into a single field.
https://plano.library.ualberta.ca/files/dn39x2787

**Comment by sfarnel** [comment]
I think I see what has happened in this case.

In old ERA isVersionOf (Citation for Previous Publication) was not
repeatable in the deposit form except for theses.

In new ERA the deposit form, which is used for everything but theses, was
made to be like the old ERA deposit form and not allow multiple instances
of this element.

When theses that did have multiple cases were transformed for migration,
the multiple instances were maintained. In migration it looks like they
have been concatenated so that the system will accept them based on
conformance to the model.

From the strict metadata point of view, separate triples for for each
instance would be ideal, although concatenation into a single one is ok
with some character as a separator.

We could generate a count using the transformed foxml of how many items are
affected (it will be all ETDs) to get a sense of the issue.

@leahvanderjagt @weiweishi @pbinkley @anayram what do you think?

On Thu, Mar 31, 2016 at 3:23 PM, Mariana Paredes-Olea <
notifications@github.com> wrote:

> While testing oai I found that the field dcterms:isVersionOf, which is
> repeatable in old ERA/thesisdeposit, is not repeatable in new ERA.
>
> @weiweishi <https://github.com/weiweishi> @pbinkley
> <https://github.com/pbinkley> do you know if the data was concatenated
> into a single field in Fedora (which would require adding to the list of
> audit changes) or if it is only a problem with the way data is presented in
> the public view form?
>
> Example: the item below comes with multiple isVersionOf fields but data is
> lumped into a single field.
> https://plano.library.ualberta.ca/files/dn39x2787
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/1037>
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by leahvanderjagt** [comment]
I agree that we should determine the extent of the problem. Also, we should
anticipate more of these. It will always be the case that some ETDs will
have multiple instances of this element.

I also agree that these instances should be separated. I don't have access
to Plano (am home sick) to see the example but I trust Sharon's
recommendations.

On Fri, Apr 1, 2016 at 8:13 AM, Sharon Farnel <notifications@github.com>
wrote:

> I think I see what has happened in this case.
>
> In old ERA isVersionOf (Citation for Previous Publication) was not
> repeatable in the deposit form except for theses.
>
> In new ERA the deposit form, which is used for everything but theses, was
> made to be like the old ERA deposit form and not allow multiple instances
> of this element.
>
> When theses that did have multiple cases were transformed for migration,
> the multiple instances were maintained. In migration it looks like they
> have been concatenated so that the system will accept them based on
> conformance to the model.
>
> From the strict metadata point of view, separate triples for for each
> instance would be ideal, although concatenation into a single one is ok
> with some character as a separator.
>
> We could generate a count using the transformed foxml of how many items are
> affected (it will be all ETDs) to get a sense of the issue.
>
> @leahvanderjagt @weiweishi @pbinkley @anayram what do you think?
>
> On Thu, Mar 31, 2016 at 3:23 PM, Mariana Paredes-Olea <
> notifications@github.com> wrote:
>
> > While testing oai I found that the field dcterms:isVersionOf, which is
> > repeatable in old ERA/thesisdeposit, is not repeatable in new ERA.
> >
> > @weiweishi <https://github.com/weiweishi> @pbinkley
> > <https://github.com/pbinkley> do you know if the data was concatenated
> > into a single field in Fedora (which would require adding to the list of
> > audit changes) or if it is only a problem with the way data is presented
> in
> > the public view form?
> >
> > Example: the item below comes with multiple isVersionOf fields but data
> is
> > lumped into a single field.
> > https://plano.library.ualberta.ca/files/dn39x2787
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub
> > <https://github.com/ualbertalib/HydraNorth/issues/1037>
> >
>
>
>
> --
> Sharon Farnel
> Metadata Coordinator
> University of Alberta Libraries
> sharon.farnel@ualberta.ca
> 780-492-3685
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/1037#issuecomment-204412477>
>



-- 
___________________________________________________________




*Leah VanderjagtDigital Repository Services Librarian University of
Albertat. 780.492.3851 *
*leahv@ualberta.ca <leahv@ualberta.ca> *

**Comment by anayram** [comment]
Generating count now

**Comment by sfarnel** [comment]
brilliant; thanks!

On Fri, Apr 1, 2016 at 9:07 AM, Mariana Paredes-Olea <
notifications@github.com> wrote:

> Generating count now
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/1037#issuecomment-204432099>
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by anayram** [comment]
Sorry for dropping the ball here! Here are the results:

There are approximately 451 objects migrated into ERA with two or more isVersionOf values
Also recently unembargoed items include ~250 objects that fit that same result.

**Comment by sfarnel** [comment]
Thanks Mariana. Good to be able to sort this as part of the audit.

On Mon, Apr 4, 2016 at 3:36 PM, Mariana Paredes-Olea <
notifications@github.com> wrote:

> Sorry for dropping the ball here! Here are the results:
>
> There are approximately 451 objects migrated into ERA with two or more
> isVersionOf values
> Also recently unembargoed items include ~250 objects that fit that same
> result.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/1037#issuecomment-205507586>
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by anayram** [comment]
@sfarnel @leahvanderjagt @sfbetz 

there are a couple of non-theses items in ERA where multiple citations were captured in a single isVersionOf field.  Just wondering if this field should be set as repeatable for all cases in ERA.

**Comment by leahvanderjagt** [comment]
As mentioned elsewhere, the decision is to keep this as a single field for
now, separate the citations in the field, and introduce a line break in
display.

On Tue, Apr 12, 2016 at 3:34 PM, Mariana Paredes-Olea <
notifications@github.com> wrote:

> @sfarnel <https://github.com/sfarnel> @leahvanderjagt
> <https://github.com/leahvanderjagt> @sfbetz <https://github.com/sfbetz>
>
> there are a couple of non-theses items in ERA where multiple citations
> were captured in a single isVersionOf field. Just wondering if this field
> should be set as repeatable for all cases in ERA.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/1037#issuecomment-209112871>
>



-- 
___________________________________________________________




*Leah VanderjagtDigital Repository Services Librarian University of
Albertat. 780.492.3851 *
*leahv@ualberta.ca <leahv@ualberta.ca> *

**Comment by leahvanderjagt** [comment]
Change of decision: this should be a repeatable field.
